src/server/DeviceServer.js

```diff
@@ -347,6 +347,7 @@ class DeviceServer {
   _onDeviceReady = async (device: Device): Promise<void> => {
     try {
       logger.log('Device online!');
+      let uuid;
```

Comment:
This definition should be closer to where it's set.
I think it should also be initialized to `null`